apps/nowcasting-app/components/layout/header/profile-dropdown.tsx

```diff
@@ -13,6 +13,14 @@ interface IProfileDropDown {}
 
 const ProfileDropDown = ({}: IProfileDropDown) => {
   const { user } = useUser();
+  const [show4hView, setShow4hView] = useGlobalState("show4hView");
+  const toggle4HourForecastVisibility = () => {
+    if (show4hView === false) {
```

Comment:
For a simple toggle function like this, can be shortened just to something like: `() => setShow4hView(!show4hView)` and can either go into a wrapper function like you have here, or if it's simple can just go inline in the onClick handler.

Reply:
thanks for showing me how to make these simpler!